src/views/AddItem.css

```diff
@@ -0,0 +1,18 @@
+.error-message {
```

Comment:
I'm wondering if this css, which seems to be repeated in several places, can be moved to index.css or layout so it only appears in one place.  Not sure about best practices with this file structure.

There's a red in :root, `--color-red`, referencing it would allow us to change every error message color from one place.